File: library/src/main/java/com/example/library/dto/response/LoanSlipSyncResponse.java
package com.example.library.dto.response;

import java.time.LocalDate;
import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;


@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class LoanSlipSyncResponse {

	   private Integer id;

	    private String receiptNumber;
	    
	    private Integer accountId;
	    
	    private Integer bookId;
	    
	    private Integer memberId;
	        
	    private Integer state;
	    
	    private Long version;

	    private LocalDate borrowDate;
	    
	    private LocalDate dueDate;
	    
	    private LocalDateTime updatedAt;
	    
	    private LocalDateTime deleteAt;
}

File: MyApplication/app/src/main/java/com/example/myapplication/data/remote/dto/response/LoanSlipSyncResponse.kt
package com.example.myapplication.data.remote.dto.response

import java.time.LocalDate
import java.time.LocalDateTime

data class LoanSlipSyncResponse(
    val id: Int,
    val receiptNumber: String,
    val accountId: Int,
    val bookId: Int,
    val memberId: Int,
    val state: Int,
    val version : Long,
    val borrowDate: LocalDate,
    val dueDate: LocalDate,
    val updatedAt: LocalDateTime,
    val deleteAt: LocalDateTime?
)